src/transformers/models/bridgetower/modeling_bridgetower.py

```diff
@@ -57,6 +58,10 @@
 ]
 
 
+def contrastive_loss(logits: torch.Tensor, labels: torch.Tensor) -> torch.Tensor:
+    return nn.functional.cross_entropy(logits, labels)
```

Comment:
Since it's just one line, let's not define a function for this and just use cross entropy.